.ci/windows/msbuild.bat

```diff
@@ -5,4 +5,4 @@ echo "Prepare context for VsDevCmd.bat"
 call "C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional\Common7\Tools\VsDevCmd.bat"
 nuget restore -verbosity detailed -NonInteractive
 
-msbuild /p:Configuration=Release
```

Comment:
Do we go back to building debug here? If so, why? I think you changed this not long ago.

Reply:
Changed this back to the previously successful test run state, as tests were hanging in CI for Windows .NET Framework when built in Release configuration. Having changed this back, the tests did still hang on a run when compiling in Debug configuration, so building in Release configuration may not be an issue. I'll open a separate issue to track because tests hanging is a more general problem to investigate.